docs/articles/sdk/read-system-logs.rst

```diff
@@ -0,0 +1,99 @@
+:orphan:
+
+Reading System Logs with DLT
+============================
+
+Introduction
+------------
+
+PELUX uses a component from GENIVI called `DLT`_ (for Diagnostic Log and Trace)
+which enables distant computers to access the system logs of a machine running
+PELUX.
+
+.. _`DLT`: https://at.projects.genivi.org/wiki/display/PROJ/Diagnostic+Log+and+Trace
+
+DLT is split into two parts:
+
+- **dlt-daemon**: running on PELUX, collecting logs and distributing them on
+  a network
+- **dlt-viewer**: running on a developer machine, accessing distant logs from
+  dlt-daemon
+
+Compiling dlt-viewer
+--------------------
+
+Since dlt-daemon is already present on PELUX, you only need to compile
+dlt-viewer on your development machine. For that, you need to have `git`,
+`cmake` and the Qt SDK installed on your machine.
+
+Then, clone the dlt-viewer repository:
+
+.. code-block:: bash
+
+    git clone https://github.com/GENIVI/dlt-viewer
+
+Compile it with cmake:
+
+.. code-block:: bash
+
+    mkdir build
+    cd build
+    cmake ..
```

Comment:
Can you run - -build at the same time as you generate the makefiles? If so we could shorten the build instructions. For example

Cmake -Bbuild - H. - - build 

Instead of all these instructions

Reply:
I couldn't find a way to do it with less commands than:
cmake -H. -Bbuild
cmake --build build
Which is already better than what I had wrote but it looks like --build expects the Makefile to be here and we can't use it in one command